packages/snfoundry/package.json

```diff
@@ -10,7 +10,8 @@
     "compile": "cd contracts && scarb build",
     "format": "prettier --write 'scripts-ts/**/*.{ts,tsx}' && cd contracts && scarb fmt",
     "format:check": "prettier --check 'scripts-ts/**/*.{ts,tsx}' && cd contracts && scarb fmt --check",
-    "postinstall": "shx cp -n .env.example .env"
+    "postinstall": "shx cp -n .env.example .env",
+    "deploy:verify": "ts-node scripts-ts/verify-contracts.ts"
```

Comment:
```suggestion
    "verify": "ts-node scripts-ts/verify-contracts.ts"
```

Reply:
lets change this to only `verify`, because `deploy:verify` doesnt really do the work of `deploying and verifying`.

Reply:
- [x] changed deploy:verify to verify